cypress/integration/free/observer.spec.ts

```diff
@@ -205,49 +132,25 @@ describe("Free tier - Observer user", () => {
   describe("Manage policies page", () => {
     beforeEach(() => {
       cy.loginWithCySession("oliver@organization.com", GOOD_PASSWORD);
-      cy.visit("/policies/manage");
+      managePoliciesPage.visitManagePoliciesPage();
     });
     it("hides manage automations button", () => {
-      cy.findByText(/manage automations/i).should("not.exist");
+      managePoliciesPage.hidesButton("Manage automations");
```

Comment:
The argument gives this function a lot of utility. I wonder if there are ways we can leverage arguments to extend other functions further and reduce the overall number of page objects.

Reply:
Maybe. I know we want the page objects to stay readable, and some page objects we want to clarify if we verify an action, so maybe we can cut down page objects by using the argument somehow as a verified action or not, idk

Reply:
That's an interesting idea.